src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerActivityExtensions.cs

```diff
@@ -128,7 +128,7 @@ public static JaegerSpan ToJaegerSpan(this Activity activity)
                 operationName: activity.DisplayName,
                 flags: (activity.Context.TraceFlags & ActivityTraceFlags.Recorded) > 0 ? 0x1 : 0,
                 startTime: ToEpochMicroseconds(activity.StartTimeUtc),
-                duration: (long)activity.Duration.TotalMilliseconds * 1000,
+                duration: (long)Math.Round(activity.Duration.TotalMilliseconds * 1000, MidpointRounding.AwayFromZero),
```

Comment:
Would ticks have better precision than milliseconds?

Reply:
Yes, I will update to use it!